tasks/perform_orca_reconcile/perform_orca_reconcile.py

```diff
@@ -222,7 +233,7 @@ def generate_orphan_reports_sql() -> TextClause:
                 size_in_bytes, 
                 storage_class
             FROM 
-                orphan_reports"""
+                orphan_reports"""  # nosec
```

Comment:
Why is there a `# nosec` here? Is bandit flagging something weird? Doesn't seem like it is needed.

Reply:
String injection on sql. Even though that isn't a thing anymore.